### DISCUSSION TITLE
Move to new Github organization

### DISCUSSION THREAD

**Discussion by Dr-Electron** [iotaledger/iota-wiki#460] [Ideas]
The Wiki with the latest developments is going to be way more than just a wiki. We may have own plugins, own themes, own tutorials, own videos and so on.
So it may make sense to move away from the iota-community repo.
Wdyt?
And if we do that, what should be the name of the new org?

**Comment by kilianhln** [comment]
Especially since we create an own repository for each tutorial, a separate organization sounds like a good idea. Maybe we should involve Antonio in this discussion ? For the Name: I am happy with [iota-wiki ](https://github.com/iota-wiki), looks like it is not taken so far.

**Comment by jlvandenhout** [comment]
Yes let's not complicate it, just call it `iota-wiki`. Let's list the things we need to think about for a successful transfer:

- CNAME registration
- Docusaurus config
- ...

**Reply by Dr-Electron** [reply]
- CNAME registration? Don't we just change the repo settings?
- In Docusaurus we should just need to change the following:
    - [organizationName](https://github.com/iota-community/iota-wiki/blob/8ae5195c3a59fdb4ce1d746d49e312236526b158/docusaurus.config.js#L13)
    - We should probably generalise [this](https://github.com/iota-community/iota-wiki/blob/8ae5195c3a59fdb4ce1d746d49e312236526b158/docusaurus.config.js#L447). Why don't we leave it default?
    - Maybe we an also use a more configurable/general solution [here](https://github.com/iota-community/iota-wiki/blob/8ae5195c3a59fdb4ce1d746d49e312236526b158/docusaurus.config.js#L474)?

**Reply by Dr-Electron** [reply]
Btw the only reason I thought about not calling it `iota-wiki` is that I don't like https://github.com/iota-wiki/iota-wiki 😅

**Reply by jlvandenhout** [reply]
> * CNAME registration? Don't we just change the repo settings?

Just listing things that came to mind, not necessarily relevant. But afaik we publish to GitHub pages right? So if we change the organization, the GitHub pages location changes as well, which is why I thought the CNAME must also be configured to resolve to the new location.

>   * We should probably generalise [this](https://github.com/iota-community/iota-wiki/blob/8ae5195c3a59fdb4ce1d746d49e312236526b158/docusaurus.config.js#L447). Why don't we leave it default?

I think this is a leftover from an earlier version of the Editor, by now I changed it to default to the organization name as commented above the setting, so we could remove it indeed.

>   * Maybe we an also use a more configurable/general solution [here](https://github.com/iota-community/iota-wiki/blob/8ae5195c3a59fdb4ce1d746d49e312236526b158/docusaurus.config.js#L474)?

Not sure how though, it needs to be set, else Docusaurus removes the edit link altogether and we can't reference the organization and repo names here, so I think it has to be explicit.

**Reply by jlvandenhout** [reply]
> Btw the only reason I thought about not calling it `iota-wiki` is that I don't like https://github.com/iota-wiki/iota-wiki 😅

But when we move to the `iota-wiki` organization it makes more sense to me to rename the repo anyways. If we are going to refactor the repo into several logical repos, we'd call them `iota-wiki/build`, `iota-wiki/docs` (or `iota-wiki/content`?), `iota-wiki/graphics`, `iota-wiki/theme`, etc. for example.

**Reply by Dr-Electron** [reply]
Which repo would contain the main stuff tho? We would always need a iota-wiki/iota-wiki repo, right?
Hmmm, o maybe iota-wiki/core. Back to fancy names it is. I'm happy :trollface:

**Reply by jlvandenhout** [reply]
If it makes you happy, I'm happy! :satisfied:

**Comment by Dr-Electron** [comment]
Me wanting fancy names.
Jeroen: 🔨 

:trollface:

**Comment by Dr-Electron** [comment]
Final decision was made  on the wiki call in IOTA discord today. We will move to the https://github.com/iota-wiki organization.
ToDo:
- [ ] Update the wiki cli
- [ ] Update the docusaurus config
- [ ] Move
